guides/InstallationAdminGuide.md

```diff
@@ -75,22 +76,29 @@ openEQUELLA uses the default Users Tablespace that must have an <unlimited> Quot
 ### Create a Microsoft SQL Server database instance for openEQUELLA
 
 If you are using Microsoft SQL Server 2008, 2012 or 2014, please ensure that the TCP/IP protocol has been enabled. The openEQUELLA user (the installer default value is ‘equellauser’) must have the following:
-* Permissions: to create, modify and delete tables, indexes and constraints and to run, select, insert, delete and update queries. 
 * Database Name: the installer default name is ‘equella’.
 * Database Role: the database user must be the database owner. For the openEQUELLA user login select ‘db_owner’.
 
 It is required that Microsoft SQL Server databases have READ_COMMITTED_SNAPSHOT enabled to avoid possible deadlocks. See http://msdn.microsoft.com/en-us/library/ms173763.aspx for more information. The following statement will enable this setting for a given database:
 
+```
 ALTER DATABASE MyEquellaDatabase SET READ_COMMITTED_SNAPSHOT ON;
+```
 
 ### Create a PostgreSQL database instance for openEQUELLA
 
 Installations using a PostgreSQL (8.0 or higher) database must create an openEQUELLA database before installing openEQUELLA. The openEQUELLA user (in the default install this is ‘equellauser’) must have the following:
-* Permissions: to create, modify and delete tables, indexes and constraints and to run, select, insert, delete and update queries. 
 * Database Name: the default name used is ‘equella’.
 * Database Owner: the default name used is ‘equellauser’.
 * Tablespace:‘pg_default’.
 
+These requirements are met by the following commands
+
+```
```

Comment:
Add code block type:
```` ```bash ````

Reply:
I used 'sh', hopefully highlights the same but happy to change to 'bash' should it be required.